tests/emscripten_main_loop_and_blocker.cpp

```diff
@@ -16,13 +16,17 @@ void final(void*) {
   assert(frame == 20);
 #ifdef REPORT_RESULT
   REPORT_RESULT(0);
+#else
+  exit(0);
```

Comment:
could the report result macro maybe call exit? that seems reasonable.

Reply:
Perhaps, but that would be a bigger and different change.        I couple of things to consider.

1. Maybe we sometimes want the program to continue?  Do we ever report multiple results?  I don't know
2. What happens if `REPORT_RESULT` is not defined at all... historically I think we wanted that to work.

In any case I think that is bigger question outside the scope of this one change.